.github/workflows/py-code-check.yml

```
@@ -48,30 +48,7 @@ jobs:
 
   iac-validate:
     if: inputs.run-iac-validation
-    name: IaC Validation
-    runs-on: ubuntu-latest
-    timeout-minutes: 10
-    defaults:
-      run:
-        working-directory: "./terraform"
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v3
-
-      - name: Setup Terraform
-        uses: hashicorp/setup-terraform@v1
-
-      - name: Terraform Init
-        id: init
-        run: terraform init -input=false -backend=false
-
-      - name: Terraform fmt
-        id: fmt
-        run: terraform fmt -check -recursive
-
-      - name: Terraform Validate
-        id: validate
-        run: terraform validate -no-color
+    uses: lifemote/workflows/.github/workflows/iac-validate.yml@feature/add-go-support
```

We can release new version for it.
```suggestion
    uses: lifemote/workflows/.github/workflows/iac-validate.yml@v1.3
```